src/main.ts

```diff
@@ -295,6 +315,24 @@ function handleKeyDown({ key }: KeyboardEvent): void {
   }
 }
 
+async function handleFindTrack() {
+  // Load track value
+  let trackId = trackInput.valueAsNumber;
+  let newTrack = dataset!.buildTrack(trackId);
+
+  // Check for track validity
+  if (newTrack.times.length > 0) {
```

Comment:
Track has a member function so you can call `newTrack.length()`, a bit less intrusive